app/views/inventory/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<h2>Inventory</h2>
+<%= render 'partials/flash' %>
+<% if @has_inv == true %>
+  <div class="row">
+    <div class="col-sm-6">
+      <div class="card">
+        <div class="card-body">
+          <h5 class="card-title"><%= @inventory.name %></h5>
+          <p class="card-text">Here you can manage all your foods in your <%= @inventory.name %>, it'd be easy to know what recipe you can cook or also what do you need to buy for all your recipes.</p>
+          <%= link_to 'Delete', "/inventory/#{@inventory.id}/destroy", class: "btn btn-danger" %>
+          <%= link_to 'Show', "/inventory/#{@inventory.id}", class: "btn btn-success" %>
+        </div>
+      </div>
+  </div>
+<% else %>
+  <%= form_with(model: @inventory, url: '/inventory/create', scope: :data) do |f| %>
+    <p>Looks like you don't have an inventory, let's create a new one =D</p>
+    <%# f.label(:name, 'Name:') %>
+    <%= f.text_field :name, class: "form-control", placeholder: 'How do you wanna name me?' %> <br>
+    <%= f.submit 'Save Inventory', class: "btn btn-primary" %>
+  <% end %>
```

Comment:
- [ ] Nice job implementing this logic so far in order to serve the need of your app. However, it's always nicer, safer, and cleaner to separate the logic from the views and move it to the rails helpers (I mean, that's what the helpers exist for right?) 😉

Reply:
Thank you very much for your suggestion! and yes you're right! 🚀